From f356775c400f40e3803d80a7bd295b4265959c45 Mon Sep 17 00:00:00 2001
From: Vineet Gupta <Vineet.Gupta1@synopsys.com>
Date: Mon, 29 Feb 2016 14:34:40 +0530
Subject: [PATCH] run-posix-option-group-test: replace CWD qith PWD

When running LTP Open Posix Testsuite there is lot of messages as:

| Usage: basename FILE [SUFFIX]
|
| Strip directory path and .SUFFIX from FILE
|
| *******************
| Testing
| *******************

This is because there is no $CWD defined in shell and it looks like $PWD
was intended instead.

Signed-off-by: Vineet Gupta <vgupta@synopsys.com>
Signed-off-by: Cyril Hrubis <chrubis@suse.cz>

Upstream-Status: Backport

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 testcases/open_posix_testsuite/bin/run-tests.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testcases/open_posix_testsuite/bin/run-tests.sh b/testcases/open_posix_testsuite/bin/run-tests.sh
index d4bd988..5ee5687 100755
--- a/testcases/open_posix_testsuite/bin/run-tests.sh
+++ b/testcases/open_posix_testsuite/bin/run-tests.sh
@@ -29,7 +29,7 @@ run_test_loop() {
 
 	cat <<EOF
 *******************
-Testing $(basename $CWD)
+Testing $(basename $PWD)
 *******************
 $(printf "PASS\t\t%3d" $NUM_PASS)
 $(printf "FAIL\t\t%3d" $NUM_FAIL)
-- 
2.7.4

